sdks/python/apache_beam/testing/synthetic_pipeline_test.py

```diff
@@ -111,7 +207,12 @@ def testSplitAtFraction(self):
         source, 1, 0.3)
 
   def run_pipeline(self, barrier, writes_output=True):
-    steps = [{'per_element_delay': 1}, {'per_element_delay': 1}]
+    steps = [{
+        'per_element_delay': 1
+    }, {
+        'per_element_delay': 1,
+        'splittable': True
+    }]
     args = ['--barrier=%s' % barrier, '--runner=DirectRunner',
```

Comment:
I would prefer using ```--runner=SwitchDirecrtRunner```, which will using a ```FnApiRunner``` instead of one non-portable ```DirectRunner``` if trying to test SDF. An SDF can be splittable only over fnapi. I'm also curious why it works on ```DirectRunner``` without involving SDF expansion and SDF-specific operation.
I may not have best judgement here though. @lukecwik what do you think here?

Reply:
The DirectRunner is currently doing an expansion here: https://github.com/apache/beam/blob/ccbad612a49ad1eacc5f515da6a3fe417b9cc7ce/sdks/python/apache_beam/runners/direct/direct_runner.py#L221

Reply:
Good to know. Thanks!

Reply:
The point of adding it there was mostly just to check that the plumbing and args were set up correctly and to just use it somewhere so I think it should be fine. The splitting is tested via unit tests.